.github/workflows/lib-e2e.yaml

```diff
@@ -28,7 +28,7 @@ jobs:
               - accel-config-demo
               - intel-deviceplugin-operator
           - name: e2e-qat-gnrd
-            targetjob: e2e-qat FOCUS="Mode:dpdk.*Resource:(cy|dc)" SKIP="App:(crypto-perf|compress-perf)"
+            targetjob: e2e-qat FOCUS="Mode:dpdk.*Resource:(cy|dc)" SKIP="(App:(crypto-perf|compress-perf)|Functionality:auto-reset)"
```

Comment:
for any new functionality, should we look into using labels?

Reply:
Umm, I thought I understood, but I think I didn't.
What was your point?
I changed a bit though...

Reply:
I was asking if the new labels approach would make sense here

Reply:
@mythi I think so, but it takes time to change all unless you are ok to change only qat. (+Makefile, lib-e2e.yaml)
So, can we just proceed with this and open an issue for that?

Reply:
https://github.com/intel/intel-device-plugins-for-kubernetes/issues/1862
I opened an issue. Please let me do that after this gets merged.

Reply:
So.. not for this pr.